### ISSUE TITLE
Dark style not working for docs

### ISSUE THREAD

**Issue by pelikhan** [microsoft/pxt-adafruit#1164]
https://makecode.adafruit.com/pkg/microsoft/pxt-maker-controller

![image](https://user-images.githubusercontent.com/4175913/67231115-88d87380-f3f3-11e9-9c30-f740b514e6ff.png)

**Comment by ajs256** [comment]
I agree, this is terrible. I can't seem to find where the module page's source is, so if someone can point me there, I could open a PR to fix this.
